_config.yml

```diff
@@ -40,6 +40,7 @@ exclude:
   - Rakefile
   - Gemfile
   - Gemfile.lock
+  - gems/
```

Comment:
It can be decided by the user, nothing in the repo suggests to install it with the source, but it's relatively common.